### PR TITLE
draft: remove setup.py

### PR DESCRIPTION
**Pull request by okken**
- remove setup.py and more fully complete pyproject.toml
- shift to pypi trusted publisher
- remove manifest
- speed up tox testing by building wheel once

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+[project.urls]
+Home = "https://github.com/pytest-dev/pytest-repeat"
+
+[project.entry-points.pytest11]
+repeat = "pytest_repeat"
+
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel", "setuptools_scm"]
-build-backend = 'setuptools.build_meta'
+requires = ["setuptools>=40.8.0", "setuptools_scm"]
```
Comment (RonnyPfannschmidt) [inline pyproject.toml:37]:
```suggestion
requires = ["setuptools>=60.0", "setuptools_scm>=8.0.0"]
```

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[tool.setuptools]
```
Comment (RonnyPfannschmidt) [inline pyproject.toml:42]: Might want to add the pypropject fmt pre commit Hook

File: tox.ini [hunk 3]
```diff
@@ -1,8 +1,3 @@
-# Tox (https://tox.readthedocs.io/) is a tool for running tests
-# in multiple virtualenvs. This configuration file will run the
-# test suite on all supported python versions. To use it, "pip install tox"
-# and then run "tox" from this directory.
-
 [tox]
 minversion = 3.4.0
```
Comment (RonnyPfannschmidt) [inline tox.ini:2]: Might want to bump to 4.x

Comment (okken) [reply tox.ini:2]: thanks @RonnyPfannschmidt

**Pull request closed without merging**